src/tools/pub/cmd/pub-downgrade.md

```diff
@@ -99,14 +99,23 @@ trying to find a set of versions that work with your package from what's already
 available.
 
 
-## Options {#options}
 
-The `pub downgrade` command supports the
-[`pub get` options](/tools/pub/cmd/pub-get#options).
+## Options
 
 For options that apply to all pub commands, see
 [Global options](/tools/pub/cmd#global-options).
 
+`--offline`
+: Use locally cached packages instead of accessing the network.
+
+`--no-offline`
+: Connect to the network to check for hosted dependencies. This is the 
+default setting.
+
+`--dry-run` or `-n`
+: Report what dependencies would change but would not change any.
```

Comment:
You can stick to the command help text: _Report what dependencies would change but don't change any._